TCAT/Controllers/HomeOptionsCardViewController.swift

```diff
@@ -0,0 +1,561 @@
+        if isDynamicSearchBar {
+            if searchBar.isFirstResponder {
+                return min(tableViewContentHeight() + CGFloat(searchBarHeight), maxCardHeight) + searchBarSeparatorHeight
+            } else { return CGFloat(searchBarHeight) + searchBarSeparatorHeight }
+        }
+        return min(tableViewContentHeight() + searchBarHeight, UIScreen.main.bounds.height*maxScreenCoverage) + searchBarSeparatorHeight
+    }
+    
+    func updatePlaces() {
+        recentLocations = SearchTableViewManager.shared.retrievePlaces(for: Constants.UserDefaults.recentSearch)
+        favorites = SearchTableViewManager.shared.retrievePlaces(for: Constants.UserDefaults.favorites)
+    }
+    
+    func animateInInfoButton() {
+        UIView.animate(withDuration: 0.1) {
+            self.infoButton.alpha = 1
+            
+            self.searchBar.snp.remakeConstraints { (make) in
+                make.leading.top.equalToSuperview()
+                make.trailing.equalTo(self.infoButton.snp.leading)
+                make.height.equalTo(self.searchBarHeight)
+            }
+            self.view.layoutIfNeeded()
+        }
+    }
+    
+    func animateOutInfoButton() {
+        UIView.animate(withDuration: 0.2) {
+            self.infoButton.alpha = 0
+            self.searchBar.snp.remakeConstraints { (make) in
+                make.leading.top.equalToSuperview()
+                make.trailing.equalTo(self.infoButton.snp.trailing)
+                make.height.equalTo(self.searchBarHeight)
+            }
+            self.view.layoutIfNeeded()
+        }
+    }
+    
+    @objc func presentFavoritesTVC(sender: UIButton? = nil) {
+        let favoritesTVC = FavoritesTableViewController()
+        let navController = CustomNavigationController(rootViewController: favoritesTVC)
+        present(navController, animated: true, completion: nil)
+    }
+    
+    /* Open information screen */
+    @objc func openInformationScreen() {
+        let informationViewController = InformationViewController()
+        let navigationVC = CustomNavigationController(rootViewController: informationViewController)
+        present(navigationVC, animated: true)
+    }
+    
+    /* Get Search Results */
+    @objc func getPlaces(timer: Timer) {
+        let searchText = (timer.userInfo as! [String: String])["searchText"]!
+        if !searchText.isEmpty {
+            Network.getSearchResults(searchText: searchText).perform(withSuccess: { response in
+                self.searchResultsSection = Section(type: .searchResults, items: response.data)
+                self.sections = [self.searchResultsSection]
+            })
+        } else {
+            sections = createSections()
+        }
+    }
+}
+
+// MARK: VC Life Cycle setup
+extension HomeOptionsCardViewController {
+    override func loadView() {
+        let customView = RoundShadowedView()
+        customView.addRoundedCornersAndShadow(radius: 10, shadowOpacity: 0.4)
```

Comment:
Can RoundShadowedView have these numbers hard-coded in the class? Or is this used with different numbers elsewhere?